skimage/feature/_kb.py

```diff
@@ -0,0 +1,264 @@
+from __future__ import print_function, division
+import numpy as np
+from scipy import ndimage as ndi
+from ..util import img_as_ubyte
+from .._shared.utils import assert_nD
+
+
+def kb(image, min_scale=10, max_scale=25, saliency_threshold=0.6, clustering_threshold=7):
+    """Finds salient regions in the given grayscale image.For each point x
+    the method picks a scale s and calculates salient score Y(x,s).
+    By comparing Y(x,s) of different points x the detector can rank
+    the saliency of points and pick the most representative ones.
+
+    Parameters
+    ----------
+    image : ndarray
+        Input grayscale image
+    min_scale : int, optional
+        Minimum scale for the keypoints detected.
+    max_scale : int, optional
+        Maximum scale for the keypoints detected.
+    saliency_threshold : float, optional.
+        Features with saliency score satisfying threshold will be considered.
+    clustering_threshold : int, optional
+        Variance among the regions should be smaller than this threshold
+        for sufficient clustering.
+
+    Returns
+    -------
+    A : (n, 3) ndarray
```

Comment:
@soupault you are absolutely right! Oops. Dunno where I got that other idea... =\

@parulsethi sorry about this mistake! But, at least, @soupault agree that the N should be capitalised. =)